cmd/kubeadm/app/util/config/cluster.go

```diff
@@ -59,7 +59,7 @@ func FetchInitConfigurationFromCluster(client clientset.Interface, w io.Writer,
 
 // getInitConfigurationFromCluster is separate only for testing purposes, don't call it directly, use FetchInitConfigurationFromCluster instead
 func getInitConfigurationFromCluster(kubeconfigDir string, client clientset.Interface, newControlPlane bool) (*kubeadmapi.InitConfiguration, error) {
-	// TODO: This code should support reading the MasterConfiguration key as well for backwards-compat
+	// TODO: This code should support reading the ClusterConfiguration key as well for backwards-compat
```

Comment:
please remove this comment.
the `MasterConfiguration` key no longer exists in our code base.